sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -167,61 +167,115 @@ ConfigurationAsyncClient client = new ConfigurationClientBuilder()
         .connectionString(appConfigConnectionString)
         .buildAsyncClient();
 
-client.listSettings(new SettingSelection().label(periodicUpdateLabel))
+client.listConfigurationSettings(new SettingSelection().label(periodicUpdateLabel))
     .subscribe(setting -> updateConfiguration(setting));
 ```
 
 ## Examples
 
 The following sections provide several code snippets covering some of the most common Configuration Service tasks, including:
-- [Create a Configuration Setting](#create-a-Configuration-Setting)
-- [Retrieve a Configuration Setting](#retrieve-a-Configuration-Setting)
-- [Update an existing Configuration Setting](#update-an-existing-Configuration-Setting)
-- [Delete a Configuration Setting](#delete-a-Configuration-Setting)
+- [Create a Configuration Setting][sample_hello_world]
```

Comment:
These should be sentence cased.
https://styleguides.azurewebsites.net/Styleguide/Read?id=2696